core/src/test/scala/akka/kafka/scaladsl/IntegrationSpec.scala

```diff
@@ -67,24 +78,35 @@ class IntegrationSpec extends TestKit(ActorSystem("IntegrationSpec"))
     producer.close(60, TimeUnit.SECONDS)
   }
 
+  def produceMessages(topic: String, range: Range) = {
+    Source(range)
```

Comment:
Also, putting ```Await``` inside would be hiding a blocking operation.  For better readability, I would propose returning the ```Future``` and then doing either ```Await``` or an ```onComplete``` handler.